lib/std/array_list.zig

```diff
@@ -6,6 +6,20 @@ const mem = std.mem;
 const math = std.math;
 const Allocator = mem.Allocator;
 
+/// Shared between managed and unmanaged versions of ArrayList. Called
+/// when memory growth is necessary. Returns a capacity larger than minimum
+/// that is better according to our growth policy.
+/// In future versions of (Unmanaged)ArrayList, we should consider taking
+/// the growth policy as parameter.
+fn computeBetterCapacity(minimum_capacity: usize) usize {
```

Comment:
Note that in master branch, the logic has two inputs, not one:

 * current capacity
 * minimum capacity